setup.py

```diff
@@ -5,7 +5,7 @@
 
 setup(
     name="bioptim",
-    version="2.3.0",
+    version="3.0.0",
```

Comment:
If it is possible to read bioptim.__version__, it would be better

Reply:
Done.